commands.js

```diff
@@ -1701,6 +1706,7 @@ var commands = exports.commands = {
 			});
 		});
 	},
+	updateserverhelp: ["/updateserver - Updates the server to a newer version. Requires: Console Access"],
```

Comment:
`hotpatch` isn't `console`. All admins have `hotpatch` access, not just console admins.